homeassistant/components/tuya/number.py

```diff
@@ -234,6 +234,15 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # Smart Gardening system
+    # https://developer.tuya.com/en/docs/iot/categorysz?id=Kaiuz4e6h7up0
+    "sz": (
+        NumberEntityDescription(
+            key=DPCode.BRIGHT_VALUE,
+            translation_key="time_since_start",
```

Comment:
This seems incorrect, according to the specification of Tuya, this is the brightness of the light:

![CleanShot 2024-05-29 at 08 39 21@2x](https://github.com/home-assistant/core/assets/195327/8d690c72-82fe-4e52-a5c5-d9f173eed440)

Reply:
In my device, it use this value to show the "Plant days" value (see app screenshot) not sure if this is very specific to my device.

Anyway I'll revert it as per Tuya spec.